### PR TITLE
Add Aspect module

### LINKED ISSUE #446: Add new module focused on groups of correlated variables
With Aspect Importance and Triplot explanations

### LINKED ISSUE #463: note adding the `scipy` dependency in NEWS.md and potentialy lower it's required version

### PR DESCRIPTION
**Pull request by arturzolkowski**
closes #446 closes #463

### CODE REVIEW

File: python/dalex/dalex/aspect/checks.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            "Association value outside the range [0, 1]. Truncated to the range."
+        )
+
+    if value < 0:
+        value = 0
+
+    if value > 1:
+        value = 1
+
+    return value
```
[the remark this answers is not in the document]
Comment (hbaniecki) [reply python/dalex/dalex/aspect/checks.py:48]: Ok. I am only wondering if (a) the user should be worried about these values being out of the `[0, 1]` range, (b) can the user do something about it? 

Maybe the warning can be more informative for the user. If not, then we can resolve this comment.

**Review by hbaniecki — APPROVED** [review]
LGTM

**Review by hbaniecki — COMMENTED** [review]
fix tests
image

**Comment by hbaniecki** [comment]
EMA snippets were tested and there are no code regressions. 

A few typos were noted in https://github.com/pbiecek/ema/issues/55, https://github.com/pbiecek/ema/issues/56, and https://github.com/pbiecek/ema/issues/57.

Thanks!

**Comment by pbiecek** [comment]
good news

**Pull request merged**